colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -420,6 +420,7 @@ object Connection {
 
 object ContentType {
   val TextPlain       = "text/plain"
+  val TextPlainUtf8   = "text/plain; charset=utf-8"
```

Comment:
hey @lxohi! would you mind updating `utf-8` -> `UTF-8` to be better aligned with the value definition [here](https://tools.ietf.org/html/rfc3629#section-8)?